docs/network-flow-visibility.md

```diff
@@ -40,6 +40,18 @@
       - [Flow Records](#flow-records)
       - [Node Throughput](#node-throughput)
       - [Network Policy](#network-policy)
+- [ClickHouse-Grafana Flow-visibility Solution](#clickhouse-grafana-flow-visibility-solution)
```

Comment:
this doesn't go in the `Flow Collectors` section?

Reply:
moved.